### PR TITLE
Translation of faq.md by iSnip

### PR DESCRIPTION
**Pull request by iSnip**
this is the rewritten version with changes suggested by Cubik65536

### CODE REVIEW

**Comment by Cubik65536** [comment]
I’ll do the review when I have some spare time

**Comment by Nofated095** [comment]
I think the translation is great. Better than mine in #15.

Nofated
________________________________
发件人: Restent Ou ***@***.***>
发送时间: Monday, September 26, 2022 7:31:41 PM
收件人: LittleSkinCommspt/LittleSkinManualDev ***@***.***>
抄送: Fuck! You! Fuūúü?uck! ***@***.***>; Assign ***@***.***>
主题: Re: [LittleSkinCommspt/LittleSkinManualDev] Retranslated faq.md by iSnip (PR #21)


@Restent approved this pull request.

Can normally read in this review
Wait @Cubik65536<https://github.com/Cubik65536> review this translate

―

**Review by Nofated095 — APPROVED** [review]

**Comment by iSnip** [comment]
any suggestions regarding to the translation :?

**Comment by Cubik65536** [comment]
@Restent Can we have a preview deployment?

File: en/faq.md [hunk 1]
```diff
@@ -6,110 +6,109 @@ sidebarDepth: 2
...
 
-## 站点使用相关
+## Site usage related
 
-### 收不到验证邮件？
+### Can't receive the verification email?
 
-请先检查你的邮箱的垃圾箱。如果垃圾箱中也没有验证邮件，尝试再发送一次验证邮件。
+Please check your emails in the "Trash" folder first. If there is still no verification email in the "Trash" folder either, try to send the verification email again.
+If you still cannot receive the verification email, please check whether the email address bound to your account is correct, if not, change it to the correct email address.
```
Comment (Cubik65536) [inline en/faq.md:14]: Blank line before

```suggestion

If you still cannot receive the verification email, please check whether the email address bound to your account is correct, if not, change it to the correct email address.
```

**Review by Cubik65536 — CHANGES_REQUESTED** [review]
Check and fix markdown syntax problems. You should have a blank line before new text lines.

**Review by Cubik65536 — APPROVED** [review]

**Comment by iSnip** [comment]
I am really dumb.

**Comment by Nofated095** [comment]
我认为可以合并。我会把 #15 重新打开，处理一下冲突，之后那个 pr 就用来修改一些其他内容了
________________________________
发件人: Cubik ***@***.***>
发送时间: Monday, September 26, 2022 9:38:39 PM
收件人: LittleSkinCommspt/LittleSkinManualDev ***@***.***>
抄送: Fuck! You! Fuūúü?uck! ***@***.***>; Mention ***@***.***>
主题: Re: [LittleSkinCommspt/LittleSkinManualDev] Retranslated faq.md by iSnip (PR #21)


I'll merge this if nobody have objections.

@Restent<https://github.com/Restent> @jinzhijie<https://github.com/jinzhijie> @Nofated095<https://github.com/Nofated095>

―

**Comment by Restent** [comment]
Agree merge. We can do it later if anyone wanna do some change.

**Pull request merged**